app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: users
+#
+#  id                 :bigint           not null, primary key
+#  email              :string(320)
+#  encrypted_password :string           default(""), not null
+#  first_name         :string(50)
+#  gender             :enum             default("other"), not null
+#  last_name          :string(50)
+#  provider           :string           default("email"), not null
+#  tokens             :json
+#  uid                :string           default(""), not null
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_email             (email) UNIQUE
+#  index_users_on_uid_and_provider  (uid,provider) UNIQUE
+#
+class User < ActiveRecord::Base
+  devise :database_authenticatable, :registerable, :validatable
+  include DeviseTokenAuth::Concerns::User
+
+  enum gender: { female: 'female', male: 'male', other: 'other' }
```

Comment:
Today, in a meeting, you explain me why you use it. I think is fantastic that you are try to test differents gems. However, i want to let you know here that we usually represent enums in the next way:
```
enum gender : {
 female: 0,
 male: 1,
 other: 2
}
```
and in the migration we add the next line:
```
t.integer :gender, null: false, default: 0
```
I don't know that gem but i suppose this classic implementation of an enum has a better database performance when you try to do a query.